app/client/templates/admin/submissions.html

```diff
@@ -1,11 +1,17 @@
 <template name="submissions">
   <div class="container">
-    <table class="striped js-sortable" data-sortlist="[[3,1]]">
+    <table class="striped js-sortable" data-sortlist="[[2,1]]">
       <thead>
         <tr>
+            <th data-field="folio">{{_ "application.folio.label"}}</th>
             <th data-field="name">{{_ "application.name.label"}}</th>
-            <th data-field="project">{{_ "application.projectTitle.label"}}</th>
+            <th data-field="project" data-empty="bottom">{{_ "application.projectTitle.label"}}</th>
```

Comment:
@varffs it tells the sorting system what the 'default' value is when its empty
ex, in line 11 `data-empty="zero"`, when rating is empty the sorting will assume is zero